galite-core/src/main/kotlin/org/kopi/galite/form/VField.kt

```diff
@@ -1864,21 +1863,31 @@ abstract class VField protected constructor(width: Int, height: Int) : VConstant
    * Checks that field value exists in list
    */
   internal open fun enumerateValue(desc: Boolean) {
-    TODO()
-    /*var value: Any? = null
-    val qrybuf: String = " SELECT " + list!!.getColumn(0).column +
-            " FROM " + evalListTable() +
-            (if (isNull(block!!.activeRecord)) "" else " WHERE " + list!!.getColumn(0).column +
-                    (if (desc) " > " else " < ").toString() + getSql(block!!.activeRecord)).toString() +
-            " ORDER BY 1" + if (desc) "" else " DESC"
+    var value: Any? = null
+    val table = evalListTable()
+    val column = list!!.getColumn(0).column
+
+    val condition = if (desc) {
+      Op.build { list!!.getColumn(0).column!! greater getSql(block!!.activeRecord).toString() }
+    } else {
+      Op.build { list!!.getColumn(0).column!! less getSql(block!!.activeRecord).toString() }
+    }
+
+    val orderBy = if (desc) SortOrder.ASC else  SortOrder.DESC
+
+    val query = if (isNull(block!!.activeRecord)) {
```

Comment:
i can't find this condition in kopi so can you remove condition & orderby then just add thise code : 
```
  val query = if (desc) {
      table.slice(column!!).select(list!!.getColumn(0).column!! greater getSql(block!!.activeRecord).toString() ).orderBy(column to SortOrder.DESC)
    } else {
      table.slice(column!!).select(list!!.getColumn(0).column!! less getSql(block!!.activeRecord).toString() ).orderBy(column to SortOrder.ASC)
    }
```

Reply:
this condition exists in kopi in line 2281